app/javascript/components/organisms/StatusModalWrapper.js

```diff
@@ -50,59 +53,72 @@ const StatusModalWrapper = ({
-              {!values.status && meta.touched && meta.error && <small className='error'>{meta.error}</small>}
-            </>
-          )}
-        </Field>
+        <FormGrid.Row paddingBottom>
+          <FormGrid.Item fullWidth>
+            <Field
+              name='status'
+              validate={validate.isPresent('Status is required')}
+              component={({ input: { onChange }, meta, ...rest }) => {
+                const hasError = !values.status && meta.touched && meta.error
+                return (
+                  <div className={classNames('form-group', hasError && 'error')}>
+                    <Label label='Status' />
+                    <StatusDropdown
+                      status={values.status}
+                      onChange={val => {
+                        onChange(val)
+                        changeFieldValue('subStatus', null)
+                      }}
+                      expand
+                      size='small'
+                    />
+                    <FieldError meta={meta} />
+                  </div>
+                )
+              }}
+            />
+          </FormGrid.Item>
+        </FormGrid.Row>
         {((values.status) && LEASE_UP_SUBSTATUS_OPTIONS[values.status]) && (
-          <Field name='subStatus'>
-            {/*
-              Inline field due to the nested level throwing error in react-final-form
-              and record level validation so it can be conditional
-            */}
-            {({ input: { onChange }, meta }) => (
-              <>
-                <h2 className={`form-label ${!values.subStatus && meta.touched && meta.error ? 'error' : ''}`}>Status Detail (required)</h2>
-                <StatusDropdown
-                  status={values.status}
-                  subStatus={values.subStatus}
-                  prompt={'Select One...'}
-                  showSubStatus
-                  onChange={onChange}
-                  dropdownClasses='margin-bottom--half'
-                  buttonClasses={['expand', 'substatus', !values.subStatus && meta.touched && meta.error ? 'error' : '']}
-                  menuClasses={['form-modal_dropdown-menu']}
-                  wrapperClasses={['margin-bottom--half', 'subStatus', !values.subStatus && meta.touched && meta.error ? 'error' : '']} />
-                {!values.subStatus && meta.touched && meta.error && <small className='error'>{meta.error}</small>}
-              </>
-            )}
-          </Field>
+          <FormGrid.Row paddingBottom>
+            <FormGrid.Item fullWidth>
+              <Field
+                name='subStatus'
+                component={({ input: { onChange }, meta }) => {
+                  const hasError = !values.subStatus && meta.touched && meta.error
+                  return (
+                    <div className={classNames('form-group', hasError && 'error')}>
+                      <Label label='Status Detail' blockNote='(required)' />
```

Comment:
could be nice to pull these into Field.js but I think they are maybe just slightly too complicated and specific for now.